### ISSUE TITLE
Code with invalid quotes get successfully converted

### ISSUE THREAD

**Issue by gousaiyang** [JarryShaw/f2format#4]
**Describe the bug**
Code with invalid quotes get successfully converted.

**To Reproduce**
```
>>> import f2format
>>> s = 'hello'
>>> code = "f'{s + ''}'"
>>> print(f2format.convert(code))
'{}'.format(s + '')
>>> eval(code)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1
SyntaxError: f-string: expecting '}'
>>>
```

**Expected behavior**
`f2format.convert()` should raise an error to report that the code is invalid

**Comment by gousaiyang** [comment]
Seems like still a bug of parso:

```
>>> import parso
>>> s = 'hello'
>>> code = "f'{s + ''}'"
>>> parso.parse(code, error_recovery=False)
<Module: @1-1>
```

**Comment by gousaiyang** [comment]
Fixed in parso 0.5.2.
